packages/ui_components_lib/lib/components/input/common_select_dropdown.dart

```diff
@@ -0,0 +1,179 @@
+import 'package:flutter/material.dart';
+import 'package:ui_components_lib/ui_components_lib.dart';
+
+/// Item of dropdown that allows displaying elements in UI.
+class CommonSheetDropdownItem<T> {
```

Comment:
Did you forget to add this widget to the storybook?

Reply:
-